packages/node/src/integrations/undici/index.ts

```diff
@@ -89,7 +93,7 @@ export class Undici implements Integration {
       const url = new URL(request.path, request.origin);
       const stringUrl = url.toString();
 
-      if (isSentryRequest(stringUrl)) {
+      if (isSentryRequest(stringUrl) || request.__sentry__ !== undefined) {
```

Comment:
I kept running into this, but I'm not sure why it's happening, the spec doesn't allow for it.

Reply:
Hmm this looks strange. As if the same request went twice through this channel? Anyway, I think it's fine to bail out in this case